store/mongo/datastore_mongo_test.go

```diff
@@ -54,7 +56,7 @@ func TestInsertWorkflows(t *testing.T) {
 
 	ctx := context.Background()
 	count, _ := testDataStore.InsertWorkflows(ctx, workflow, workflow)
-	assert.Equal(t, 1, count)
+	assert.Equal(t, 2, count)
```

Comment:
See my comment above, I don't get why you are incrementing these counters.

Reply:
only to return the number of items added. right now it returns 0 which means that 1 item was added, what if 0 were added?
somewhere in the code in comments you said: "return number of records added" or something like that. I am trying to take care so it is not a lie :)

Reply:
Ok, you are probably right and I was wrong when I wrote that code :)